Makefile

```diff
@@ -105,16 +105,21 @@ GINKGO_NOCOLOR ?= false
 ARTIFACTS ?= $(ROOT_DIR)/_artifacts
 E2E_CONF_FILE ?= $(ROOT_DIR)/test/e2e/config/e2e_conf.yaml
 E2E_CONF_FILE_ENVSUBST ?= $(ROOT_DIR)/test/e2e/config/e2e_conf_envsubst.yaml
+E2E_TEMPLATE_FILE ?= $(ROOT_DIR)/templates/test/cluster-template-prow-ha.yaml
+E2E_TEMPLATE_FILE_ENVSUBST ?= $(ROOT_DIR)/templates/test/cluster-template-prow-ha_envsubst.yaml
 SKIP_CLEANUP ?= false
 SKIP_CREATE_MGMT_CLUSTER ?= true
 
+.PHONY: e2e-substitutions
+e2e-substitutions: $(ENVSUBST)
+	$(ENVSUBST) < $(E2E_CONF_FILE) > $(E2E_CONF_FILE_ENVSUBST)
+	$(ENVSUBST) < $(E2E_TEMPLATE_FILE) > $(E2E_TEMPLATE_FILE_ENVSUBST)
```

Comment:
do we really need a separate make target for env substitution? previously we had it within the e2e-tests target is not it?